### DISCUSSION TITLE
List following user friendships status don't have isFollowingYou (currently nil)

### DISCUSSION THREAD

**Discussion by huyparody** [sbertix/Swiftagram#253] [Q&A] [answered]
Hi @sbertix 

After I request for the list user I'm following, I've stored an array of ids of them to continue request friendship status of all of them

          Endpoint.user(idsArray)
                    .friendships
                    .unlock(with: secret)
                    .session(.instagram)
                    .subscribe(on: DispatchQueue.global(qos: .background))
                    .map({$0.friendships})
                    .receive(on: RunLoop.main)
                    .sink(receiveCompletion: {_ in}) { [weak self] friendShipValue in
                        print(friendShipValue?.forEach({print($0.value)}))
                    }
                    .store(in: &self.bin)

But currently isFollowingYou is nil
Below is all it had. 
`["incomingRequest": false, "isBestie": false, "following": true, "isFeedFavorite": false, "isRestricted": false, "isPrivate": false, "outgoingRequest": false]`

So how do I get isFollowingYou status from a list user?

**Comment by sbertix** [comment, marked as answer]
Hey @huyparody,

Unfortunately you only get that value for single friendships check, not using a collection of identifiers. 
Depending of the amount of followers/following and the profiles you need to check it's usually safer and easier to fetch all followers and following and compare them.
